Formula/gammalib.rb

```diff
@@ -0,0 +1,41 @@
+class Gammalib < Formula
+  desc "Toolbox for the analysis of astronomical gamma-ray data"
+  homepage "http://cta.irap.omp.eu/gammalib/about.html"
+  url "http://cta.irap.omp.eu/ctools/releases/gammalib/gammalib-1.6.3.tar.gz"
+  sha256 "b93bc03562ba1fcb21ab6f48943f9c28230c04d1b0ef986569c88b05d8e9dbbe"
+  revision 1
+
+  depends_on "cfitsio"
+  depends_on "doxygen"
+  depends_on "ncurses"
+  depends_on "python"
+  depends_on "readline"
+  depends_on "swig"
+
+  def install
+    system "./configure", "PYTHON=/usr/local/bin/python3", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  def caveats; <<~EOS
```

Comment:
This is not specific to homebrew and should just be in the documentation.

Reply:
I'm sorry.
what do you mean?

Reply:
Don't add caveats for instructions that are not specific to homebrew, like this one.